### DISCUSSION TITLE
Select mode by name in combi mode

### DISCUSSION THREAD

**Discussion by jheidbrink** [davatorium/rofi#1906] [Q&A]
When in combi mode, I'd like to find entries by the mode name. For example, when running
`rofi -combi-modi run,ssh -show combi`, when I type `ssh`,  the selection narrows down to `run ssh`, `run ssh-add`, `run ssh-agent` etc. Instead I'd also like to see all options provided by the ssh mode.

The motivation for this is that I want to use rofi as a universal fuzzy search interface for a lot of different usecases, and being able to discriminate by mode/plugin would be very helpful for that.

**Comment by DaveDavenport** [comment]
type [!ssh](https://davatorium.github.io/rofi/1.7.5/rofi.1/#combi) ?
